src/components/Drawer/Drawer.tsx

```diff
@@ -34,7 +34,7 @@ type EnforcedAccessibleLabel =
 
 const propsAreAccessible = (
 	title: string | JSX.Element | undefined,
-	buttons: JSX.Element[] | null | undefined,
+	actionNodes: React.ReactNode[] | undefined,
```

Comment:
nit: I think we use `actions` everywhere else
```suggestion
	actions: React.ReactNode[] | undefined,
```